ts/client/src/accounts/mangoAccount.ts

```diff
@@ -1283,6 +1277,25 @@ export class PerpPosition {
     );
   }
 
+  public getQuotePositionUi(
+    perpMarket: PerpMarket,
+    useEventQueue?: boolean,
+  ): number {
+    if (perpMarket.perpMarketIndex !== this.marketIndex) {
+      throw new Error("PerpPosition doesn't belong to the given market!");
+    }
+
+    const quotePositionUi = toUiDecimalsForQuote(this.quotePositionNative);
+
+    return useEventQueue
+      ? quotePositionUi + perpMarket.quoteLotsToUi(this.takerQuoteLots)
+      : quotePositionUi;
+  }
+
+  public getNotionalValueUi(perpMarket: PerpMarket): number {
```

Comment:
could this still pass through `useEventQueue`? Then users can use getquotePosUi/getNotationalValueUi either with or without event queue.